src/docs/development/platform-integration/platform-views.md

```diff
@@ -473,13 +473,25 @@ class FLNativeView: NSObject, FlutterPlatformView {
         arguments args: Any?,
         binaryMessenger messenger: FlutterBinaryMessenger?
     ) {
-        super.init()
         _view = UIView()
+        super.init()
```

Comment:
doesn't this need to be called first?

Reply:
either is fine but I'd just leave this first. We don't expect the super to access any overridden properties.

Reply:
Calling `super.init()` first throws this error in Xcode.

<img width="677" alt="Screen Shot 2020-11-21 at 4 42 37 PM" src="https://user-images.githubusercontent.com/4143153/99871852-a12f4900-2c18-11eb-9080-7e0a3fb9ce23.png">